TeslaMateAgile/appsettings.json

```diff
@@ -18,5 +18,8 @@
   "Awattar": {
     "BaseUrl": "https://api.awattar.de/v1",
     "VATMultiplier": 1.19
-  }
+  },
+  "Barry": {
+    "BaseUrl": "https://jsonrpc.barry.energy/json-rpc",
+  },
```

Comment:
Ditto on comma for this line too:
![image](https://user-images.githubusercontent.com/2363642/148591561-afc8ee98-534e-4545-ae6b-a9fa628de9e0.png)


```suggestion
  }
```